packages/tideline/js/tidelinedata.js

```diff
@@ -673,6 +673,11 @@ TidelineData.prototype.deduplicatePhysicalActivities = function deduplicatePhysi
   });
   // For each eventID take the most recent item
   this.physicalActivities = _.map(physicalActivities, _.head);
+  this.physicalActivities = _.remove(this.physicalActivities, (pa) => {
+    if (pa.duration.value !== 0){
+      return pa;
+    }
+  })
```

Comment:
Maybe using filter is more readable. Here I'm not sure what it does.
In one path the function return an object, and in another there is no return.

```suggestion
  this.physicalActivities = _.filter(this.physicalActivities, (pa) => pa.duration.value > 0);
```